src/Control/Lens/Indexed.hs

```diff
@@ -422,7 +423,7 @@ imapM_ f = liftM skip . getSequenced #. ifoldMap (\i -> Sequenced #. f i)
 -- @
 -- 'Control.Lens.Fold.forMOf_' l a ≡ 'iforMOf' l a '.' 'const'
 -- @
-iforM_ :: (FoldableWithIndex i t, Monad m) => t a -> (i -> a -> m b) -> m ()
+iforM_ :: (FoldableWithIndex i t, Apply m, Monad m) => t a -> (i -> a -> m b) -> m ()
```

Comment:
Looks like this needs an explicit `Applicative` constraint to make `lens` build on old GHCs, as shown in [this](https://travis-ci.org/ekmett/lens/jobs/277127377#L615) failing Travis CI run.